section1/reflection.md

```diff
@@ -1,19 +1,41 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+    - Feel good about how ask for questions I think setting a time limit on self research is a good rule of thumb. I love the idea of "rubber ducking" for the try explaining the problem to someone else with out it bing a real someone else (yet) or start typing you questions and it might prompt the Ah Ha moment. I have experienced these moment even in my current job. But if you get it typed out after doing some of your own searching it is worth asking for help.
 
 2. How would you print the string `"Hello World!"` to the terminal?
+   1. Start a `irb` session in the terminal.
+   2. Run the following
+  ```ruby
+  puts "Hello World!"
+  ```
 
 3. What character is used to indicate comments in a ruby file?
+    - `#` - the hash is the first character of the line for a comment
```

Comment:
I prefer to call this the 'pound sign' instead since 'hash' can also more commonly refer to a datatype in Ruby.